### DISCUSSION TITLE
Zeitabhängige Höhe der Durchflussrate / Time-dependent level of the flow rate

### DISCUSSION THREAD

**Discussion by digi303** [jomjol/AI-on-the-edge-device#1647] [Ideas]
Hallo,

ich habe das Problem, dass teilweise falsche Werte erkannt werden, welche dann viel zu hoch sind.
Zum Beispiel vom Vorwert 465,983 m³ auf 468,002 m³ - Klar ist, dass die 6 für eine 8 gehalten worden ist.

Da ich die Rate auf 50 Liter (bei einem  Timeintervall auf 3 Minuten - das ist der maximal mögliche Durchfluss bei mir) eingestellt habe ergibt sich nun das Problem, dass die ermittelte Rate relativ schnell zu groß wird. (sofern weiteres Wasser entnommen wird).
Ab diesem Zeitpunkt habe ich nur noch Rate too high - was ja dann auch stimmt.

Könnte man es nicht so halten, dass die Rate jeweils mit dem Zeitintervall multipliziert wird?
Sprich nach 6 Minuten max 100 Liter, nach 30 Minuten ein halber m³ etc.

So hat er dann bei richtiger Digitalisierung jedenfalls wieder einen richtigen Wert. Oder hab ich da den falschen Gedanken...

Hello,

I have the problem that sometimes wrong values are recognised, which are then much too high.
For example, from the previous value of 465.983 m³ to 468.002 m³ - it is clear that the 6 was taken for an 8.

Since I have set the rate to 50 litres (with a time interval of 3 minutes - this is the maximum possible flow rate for me), the problem arises that the determined rate becomes too high relatively quickly. (if more water is taken).
From this point on, I only have rate too high - which is also true.

Couldn't we keep it so that the rate is multiplied by the time interval?
For example, after 6 minutes max 100 litres, after 30 minutes half a m³, etc.

That way, if the digitalisation is correct, it will have a correct value again. Or have I got the wrong idea...?

Translated with www.DeepL.com/Translator (free version)

**Comment by jomjol** [comment]
This is already incorporated!

Please read the informations in the configuration page carefully and use:
![grafik](https://user-images.githubusercontent.com/30766535/208740514-78a296b6-19d9-4465-8a64-97e31912e02e.png)

**Reply by digi303** [reply]
argh, sorry.
During configuration, I had massive problems with the display of the config page (probably due to wifi reception problems), so I really overlooked this.
Thank you.
